commands/leb128.go

```diff
@@ -0,0 +1,69 @@
+package commands
+
+import (
+	"fmt"
+	"io"
+	"strconv"
+
+	"github.com/filecoin-project/go-leb128"
+	cmdkit "github.com/ipfs/go-ipfs-cmdkit"
+	cmds "github.com/ipfs/go-ipfs-cmds"
+)
+
+var leb128Cmd = &cmds.Command{
+	Helptext: cmdkit.HelpText{
+		Tagline:          "Leb128 cli enc/decode",
+		ShortDescription: `Reading and writing leb128.`,
```

Comment:
nit: say leb128 uint64s.  Then we can updated this when we add big ints to this command using a flag/subcommand.